k9mail/src/test/java/com/fsck/k9/mailstore/AttachmentResolverTest.java

```diff
@@ -0,0 +1,105 @@
+package com.fsck.k9.mailstore;
+
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
+
+import android.net.Uri;
+
+import com.fsck.k9.mail.BodyPart;
+import com.fsck.k9.mail.Multipart;
+import com.fsck.k9.mail.Part;
+import com.fsck.k9.message.extractors.AttachmentInfoExtractor;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+@SuppressWarnings("unchecked")
+@RunWith(RobolectricTestRunner.class)
+@Config(manifest = Config.NONE, sdk = 21)
+public class AttachmentResolverTest {
+    public static final Uri ATTACHMENT_TEST_URI_1 = Uri.parse("uri://test/1");
+    public static final Uri ATTACHMENT_TEST_URI_2 = Uri.parse("uri://test/2");
+
+
+    @Test
+    public void buildCidMap__onPartWithNoBody__shouldReturnEmptyMap() throws Exception {
+        AttachmentInfoExtractor attachmentInfoExtractor = mock(AttachmentInfoExtractor.class);
```

Comment:
This line is part of every test. Might as well extract it to a `setUp()` method.